globals.h

```diff
@@ -0,0 +1,24 @@
+/**
+Copyright 2024 JasmineGraph Team
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+#include <string>
+
+#ifndef JASMINEGRAPH__GLOBALS_H_
+#define JASMINEGRAPH__GLOBALS_H_
+
+#define PROFILE_NATIVE 0
```

Comment:
Why do we still have a native profile here? We should have only two profiles for JasmineGraph.

Reply:
The code still has native mode parts. Until they are removed, we need to separate them by the profile.

Reply:
Ok @thevindu-w Created [Issue 243](https://github.com/miyurud/jasminegraph/issues/243)  to keep track of this task. When we have some free time lets work on it.